docs/docsite/rst/user_guide/intro_bsd.rst

```diff
@@ -54,7 +54,7 @@ To support a variety of Unix-like operating systems and distributions, Ansible c
 .. code-block:: text
 
     [freebsd:vars]
-    ansible_python_interpreter=/usr/local/bin/python2.7
+    ansible_python_interpreter=/usr/local/bin/python3.8
```

Comment:
Can you please do the change on the line number? - https://github.com/ansible/ansible/blob/36287e98108b2d53f4813770bfe93630026c86b9/docs/docsite/rst/user_guide/intro_bsd.rst#L66